benches/osrank_naive.rs

```diff
@@ -52,9 +54,10 @@ fn construct_network_small() -> Network<f64> {
     ];
     for edge in &edges {
         network.add_edge(
+            2,
             &edge.0.to_string(),
             &edge.1.to_string(),
-            2,
+            edge.2.as_f64().unwrap(),
```

Comment:
Do you plan to remove the weight parameter in the future?

Reply:
That's a good question, and perhaps something to bring to the Graph API Working Group attention. In this new world `add_edge` comes from the "official" crate and it takes a "weight" and a "data" as parameters. Unfortunately (we) model our metadata to _also_ embed the weight into it, which is now leading to duplication. Perhaps here we should follow the WG's recommendation and treat the two entities as completely separate? 

Let's discuss 😉